modules/Discord Integration/hooks/DiscordFormatterHook.php

```diff
@@ -0,0 +1,47 @@
+<?php
+/*
+ *  Made by Partydragen
+ *  https://github.com/NamelessMC/Nameless/
+ *  NamelessMC version 2.0.0 pre-13
+ *
+ *  Discord formatter hook
+ */
+
+class DiscordFormatterHook extends HookBase {
+
+    public static function format(array $params = []): array {
+        $data = $params['data'];
+        $format = $params['format'];
+
+        if ($data['event'] == 'registerUser') {
```

Comment:
How does this work for events outside of registerUser or createAnnouncement? Can this be extended by a module to add new cases?

Reply:
Other modules can add their own event listeners like that ^^ all events have events names like that so they can format their own events or other events

i also tested this for my own modules to make my own format for my events and that worked fine just like that ^^